### PR TITLE
STAR-993 - MegaMenu adjust based on screen resolution & dependancies updated And Previous UI updated

### PR DESCRIPTION
**Pull request by sadanandl**
### Description

1. Dependencies updated.
2. Screen resolution-based menu item adjust
3. UI correction from the previous commit ID

Related PR: https://github.com/brainstormforce/astra-pro-sites/pull/1479
### Checklist:
- [ ] My code is tested
- [ ] My code follows accessibility standards 
- [ ] My code has proper inline documentation 
- [ ] I've included any necessary tests 
- [ ] I've included developer documentation 
- [ ] I've added proper labels to this pull request

### CODE REVIEW

File: .DS_Store
Comment (vrundakansara) [inline .DS_Store:1]: Let's add this in .gitignore

File: src/.DS_Store
Comment (vrundakansara) [inline src/.DS_Store:1]: Add in gitignore

**Review by vrundakansara — CHANGES_REQUESTED** [review]

File: src/.DS_Store
Comment (sadanandl) [reply src/.DS_Store:1]: Fixed

File: .DS_Store
Comment (sadanandl) [reply .DS_Store:1]: Fixed

**Review by vrundakansara — APPROVED** [review]

**Comment by kudaleganesh** [comment]
@sadanandl Linting is missing.

**Review by kudaleganesh — CHANGES_REQUESTED** [review]
@sadanandl Linting is missing.

**Review by jaieds — APPROVED** [review]

**Review by kudaleganesh — APPROVED** [review]

**Pull request merged**